pkg/runcfanotify/runcfanotify.go

```diff
@@ -309,7 +314,8 @@ func (n *RuncNotifier) watchRuncIterate() (bool, error) {
 	}
 
 	// Don't leak the fd received by GetEvent
-	defer data.Close()
+	dataFile := data.File()
+	defer dataFile.Close()
```

Comment:
Is there also a double fd close risk here when using `defer data.Close()` if `data.File()` is not used?

Reply:
No I don't think so.

`data.File()` has to be called exactly one time so that the fd is closed exactly one time.

Reply:
Right, given that in this case we're not using the file, couldn't we just use `defer data.Close()`?

Reply:
@mauriciovasquezbernal not sure if I'm missing something, but note that it is `dataFile.Close()`, not `data.Close()`.

Reply:
What I mean is that in this case the original `defer data.Close()` should've worked fine because we're not using `data.File()` in this function anyway.

Reply:
> What I mean is that in this case the original defer data.Close() should've worked fine because we're not using data.File() in this function anyway.

Correct. I changed it here too for consistency with the other case where it's needed.